doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1,4401 +1,15 @@
 :orphan:
 
 .. _known_issues:
```

Comment:
Hmm, how to modify current known issues?

I would like to remove this one which was present in 2.5.0 but was fixed now:

"KRKNWK-17864: When using Wi-Fi low power mode, the communication with the device might not work after it re-connects to the newly respawned Wi-Fi network"
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/known_issues.html#matter